strategy/live_trader.py

```
@@ -1097,11 +1103,294 @@ def restart(self):
+            r = sess.get(f"https://data-api.polymarket.com/activity?user={addr}&limit=500", timeout=(4.0, 10.0))
+            if r.ok:
+                activities = r.json()
+        except Exception as e:
+            log.warning("Could not fetch activity endpoint: %s", e)
+
+        if not isinstance(activities, list) or not activities:
+            try:
+                r = sess.get(f"https://data-api.polymarket.com/trades?user={addr}&limit=200", timeout=(4.0, 10.0))
+                if r.ok:
+                    activities = r.json()
+            except Exception as e:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)